bunker/map.conf

```
@@ -0,0 +1,17 @@
+team.1.pos = (103,-32,104)
+team.1.color = red
+team.2.pos = (-103,-32,-103)
+team.1 = red
+team.2 = blue
+team.2.color = blue
+
+initial_stuff = default:pick_stone,default:torch 10
+r = 115
+h = 140
+rotation = z
+base_node = ctf_map:stonebrick
+
+name = The Bunker In the Snow
+author = realyg,winniepee0,smugler5,Soulseeker,Darkf4antom and Tryhard_
+
+license = CC BY-SA 4.0 Thanks for j45 to host the server to create the map.
```

I think that text should be moved to somewhere else.
```suggestion
license = CC BY-SA 4.0
```

---

We want to give j45 some credits.

---

```suggestion
license = CC BY-SA 4.0
others = Thanks for j45 to host the server to create the map.
```

Will this work?

---

Yes, it should work. When you test, type /maps and select your map from the list.

---

Maybe `"Thanks to j45 ..."` would be better in English